realm/realm-library/build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    provided 'junit:junit:4.12'
```

Comment:
@zaki50 Should we create another java library and deploy it? Or is there another way?